### PR TITLE
Fix VNC for hass.io image

### PR DESCRIPTION
**Pull request by neffs**
Unfortunately there were some issues with my previous PR. Seems like everyone just tested the regular docker container. :(

This works fine on hassio locally. 

I left the -kill command as a comment, in case it might be needed in the future.

### CODE REVIEW

File: armhf-hassio-addon/run.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ HASSIO_DEBUG_OTAU="$(jq --raw-output '.debug_otau' $CONFIG_PATH)"
 HASSIO_DECONZ_WEB_PORT="$(jq --raw-output '.web_port' $CONFIG_PATH)"
 HASSIO_DECONZ_WS_PORT="$(jq --raw-output '.websockets_port' $CONFIG_PATH)"
 HASSIO_DECONZ_DEVICE="$(jq --raw-output '.deconz_device' $CONFIG_PATH)"
-HASSIO_DECONZ_VNC_MODE=":$(jq --raw-output '.vnc.active' $CONFIG_PATH)"
+HASSIO_DECONZ_VNC_MODE="$(jq --raw-output '.vnc.active' $CONFIG_PATH)"
 HASSIO_DECONZ_VNC_DISPLAY=":$(jq --raw-output '.vnc.port - 5900' $CONFIG_PATH)"
```
Comment (carlioth) [inline armhf-hassio-addon/run.sh]: what did the semicolon do? Should you remove it for the HASSIO_DECONZ_VNC_DISPLAY value aswell?

Comment (neffs) [reply armhf-hassio-addon/run.sh]: No, it is correct there. 
The VNC server is started with `tigervncserver :0` and `export DISPLAY=:0` (for port 5900)

**Comment by corvy** [comment]
Have you seen this? 

https://github.com/marthoc/docker-deconz/issues/91

**Comment by pergolafabio** [comment]
will this also fix the issue that now phoscon is running on port 80 instead of 8080 as default?
seems we cant change the port anymore

**Comment by corvy** [comment]
Yes it will.

**Comment by pergolafabio** [comment]
ok, thnx, looking forward to release .56 :)
finally able to see my mesh :-)

**Comment by corvy** [comment]
Me too!!

**Pull request merged**